install.sh

```
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+ENVIRONMENT_FILE="retrospective-update/environment.yaml"
+ENVIRONMENT_NAME="update"
+
+sudo apt-get update
+
+# Download and install Miniconda
+echo "Downloading and installing Miniconda..."
+curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
+bash Miniforge3-$(uname)-$(uname -m).sh -b
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)